src/librustc/mir/interpret/allocation.rs

```diff
@@ -629,15 +628,15 @@ impl<Tag, Extra> Allocation<Tag, Extra> {
         AllocationDefinedness { ranges, initial, }
     }
 
-    /// Apply multiple instances of the run-length encoding to the undef_mask.
+    /// Applies multiple instances of the run-length encoding to the undef mask.
```

Comment:
Keep underscore, as before, this is a field.

Reply:
Same as above.